addons/onboarding/models/onboarding_onboarding.py

```diff
@@ -48,6 +48,10 @@ def _compute_current_progress(self):
             current_progress_id = onboarding.progress_ids.filtered(
                 lambda progress: progress.company_id.id in {False, self.env.company.id})
             if current_progress_id:
+                if len(current_progress_id) > 1:
+                    last_create_date = max(current_progress_id.mapped('create_date'))
```

Comment:
You could instead use the "sorted" method on the recordset and take the first one directly?

Seems shorter.